analytic_partner/views/res_partner_views.xml

```diff
@@ -1,30 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
+<odoo>
 
-        <record model="ir.actions.act_window" id="act_analytic_cost_revenue">
-            <field name="context">{'search_default_group_date': 1, 'search_default_group_journal': 1, 'default_other_partner_id': active_id}</field>
-            <field name="domain">[('other_partner_id', 'child_of', active_id)]</field>
-            <field name="name">Costs &amp; Revenues</field>
-            <field name="res_model">account.analytic.line</field>
-            <field name="src_model">res.partner</field>
-            <field name="view_mode">tree,form</field>
-            <field name="view_type">form</field>
-        </record>
+    <record model="ir.actions.act_window" id="act_analytic_cost_revenue">
+        <field name="context">{'search_default_group_date': 1, 'search_default_group_journal': 1, 'default_other_partner_id': active_id}</field>
```

Comment:
For me, it makes no sense to add "default_other_partner_id" if the domain is  ('other_partner_id', 'child_of', active_id) because you get always the other_partner_id